dev-util/webstorm/webstorm-2017.2.3.172.3968.27.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils versionator
+
+DESCRIPTION="JavaScript IDE for client- and server-side development with Node.js"
+HOMEPAGE="http://www.jetbrains.com/webstorm"
+SRC_URI="http://download.jetbrains.com/${PN}/WebStorm-$(get_version_component_range 1-3).tar.gz"
+
+LICENSE="WebStorm WebStorm_Academic WebStorm_Classroom WebStorm_OpenSource WebStorm_personal"
+SLOT="$(get_major_version)"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=virtual/jdk-1.7"
```

Comment:
Is Java really needed here?

Reply:
yes, the entire ide for webstorm is written in java